### DISCUSSION TITLE
EdgeTX Buddy

### DISCUSSION THREAD

**Discussion by Richjanz** [EdgeTX/edgetx#3051] [General]
I wish to flash my Horus X10S Express from OpenTX to EdgeTX. On the Buddy site it asks for the radio model. I see "Horus x10" and "Horus x10 Access". With my X10S Express which one do I select?

**Comment by pfeerick** [comment]
X10S Express runs the Frsky ACCESS protocol, so you want the "Horus x10 Access". The "Horus x10" is the older ACCST-only radio.

**Comment by Richjanz** [comment]
Thank you for clearing that up for me
richjanz


-----Original Message-----
